mesonbuild/envconfig.py

```diff
@@ -207,7 +207,7 @@ def is_windows(self):
         """
         Machine is windows?
         """
-        return self.system == 'windows'
+        return self.system in ('windows', 'mingw')
```

Comment:
this really isn't what a tuple is for. Can we use a list or a set here?

Reply:
Mainly copying the style of the code a few lines below. I can change both.

Reply:
It's just a pet peeve of mine :) Lists are for homogenious data, tuples for heterogenious.